Userland/Libraries/LibWebView/ViewImplementation.h

```diff
@@ -177,6 +177,7 @@ class ViewImplementation {
     Function<void(i32, String const&, Vector<Attribute> const&)> on_inspector_replaced_dom_node_attribute;
     Function<void(i32, Gfx::IntPoint, String const&, Optional<String> const&, Optional<Attribute> const&)> on_inspector_requested_dom_tree_context_menu;
     Function<void(String const&)> on_inspector_executed_console_script;
+    Function<Array<int, 2>()> on_request_worker_agent;
```

Comment:
IMO this would be nicer as a named struct. 

We have a couple of other simple structs in LibWebView (e.g. Attribute.h) for WebContent IPC, so we could also have something like:

```c++
namespace WebView {

struct SocketPair {
    IPC::File socket;
    IPC::File fd_passing_socket;
};

}

namespace IPC {

template<>
ErrorOr<void> encode(Encoder&, WebView::SocketPair const&);

template<>
ErrorOr<WebView::SocketPair> decode(Decoder&);

}
```

Reply:
sounds good, done